index.js

```diff
@@ -251,23 +242,23 @@ This will become a hard error in the future.`,
       }
       const ms = new MagicString(code);
 
-      tippex.match(code, workerRegexpForOutput, (fullMatch, optionsWithCommaStr, optionsStr) => {
-        let options;
```

Comment:
Looks like you accidentally removed this btw, so code now overrides global `options` instead.